zyngine/zynthian_state_manager.py

```diff
@@ -938,6 +938,14 @@ def get_state(self):
         b64_data = base64.b64encode(binary_riff_data)
         state['zynseq_riff_b64'] = b64_data.decode('utf-8')
 
+        # TouchKeypad config
```

Comment:
Any state/snapshot structure changes must be documented in zynthian_state_schema.py.

Reply:
Thanks for hint!